tests/components/template/test_cover.py

```diff
@@ -240,13 +294,57 @@ async def setup_single_attribute_state_cover(
                 **extra,
             },
         )
+    elif style == ConfigurationStyle.TRIGGER:
+        await async_setup_trigger_format(
+            hass,
+            count,
+            {
+                **NAMED_COVER_ACTIONS,
+                "state": state_template,
+                **extra,
+            },
+        )
+
+
+@pytest.fixture
+async def setup_empty_action(
```

Comment:
Is this fixture really related to adding support for trigger cover?

Reply:
It's not, but the previous test didn't cover Modern style.  So I converted the test to cover Legacy, Modern, and Trigger.